app/models/owner.js

```diff
@@ -22,6 +22,10 @@ export default Model.extend({
   githubId: attr('string'),
   education: attr('boolean'),
   fullName: or('name', 'login'),
+  permissions: attr(),
+  type: attr('string'),
+  isUser: equal('type', 'user'),
+  isOrganization: equal('type', 'organization'),
```

Comment:
Oh, I like this type extraction! Probably it would also make sense to remove these properties from `user` and `organization` models.

Reply:
ooh yeah that's a good idea